src/easier68k/assembler/assembler.py

```diff
@@ -0,0 +1,48 @@
+from ..core.util.parsing import strip_comments, has_label, get_label, strip_label, get_opcode, strip_opcode, parse_literal
+import binascii
+import sys
+from ..core import opcodes
+from ..core.opcodes import *
+
+
+def parse(text: str):  # should return a ListFile and errors/warnings eventually
+    # --- PART 1: process for labels and equates ---
+
+    labels = {}
+    equates = {}
+
+    line_index = 0
+    for line in text.splitlines():
+        line_index += 1
+        stripped = strip_comments(line)
+        if not stripped.strip():
+            continue
+
+        print(strip_comments(line))
```

Comment:
I don't think that we should have this print to console by default. Let whatever case is using this package print to console.

Reply:
I concur.

Reply:
Definitely, I was just using these for testing. This won't be here for the final output.